src/OpenTelemetry/Metrics/HistogramConfiguration.cs

```diff
@@ -0,0 +1,26 @@
+// <copyright file="HistogramConfiguration.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+namespace OpenTelemetry.Metrics;
+
+public class HistogramConfiguration : MetricStreamConfiguration
```

Comment:
Do we need a separate class for this? Why not just add `RecordMinMax` to `ExplicitBucketHistogramConfiguration`?

Reply:
https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/datamodel.md#exponentialhistogram MinMax is possible for ExponentialHistogram as well. So i think a base class is right place to capture MinMax.